### DISCUSSION TITLE
无名浅观察 | /posts/untitled1/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#604] [Announcements]
浅观察是瞎编的一个词语，这么干只是不想把博客题目写作又是流水账。

https://yuanfan.rbind.io/posts/untitled1/

**Comment by botsp** [comment]
雷霆雨露、皆是天恩。这真是一种很有烙印的思维毒药了，从个人与家庭、到集体与社会，在很多地方都能看到这种痕迹。父对子、夫对妻、君对臣，有时候感觉都是一回事。

以前看张艺谋的滿城盡帶黃金甲就有一个类似的情节、"天地万物，朕赐给你，才是你的；朕不给，你不能抢。"

最近看一个知乎回答讲谷歌-清华的陈某案件，说这就是东亚文化的"暴君思维"，爱与恨都来源于神的给予。当神感觉到失控，那就是激烈的愤怒与惩罚(Beating his wife to death)。

**Reply by earfanfan** [reply]
我最近读的一本书讲了一些国家治理的制度逻辑，感觉也可以借来牵强地解释一下家庭中的父权制度逻辑。那本书讲古代中国的权威类型属于“传统权威”，比如清朝的一统体制下，皇帝就是最高权威，拥有最高级别的专断权力。建立新中国后，由于种种历史因素，“传统权威”转变为“卡理斯玛权威”，即最高统帅凭借个人魅力来征服那些忠诚于他的人，随着时间转移，也为了继续维护一统体制，政府仍然需要巩固“卡理斯玛权威”，方式就是大量宣传最高统帅的个人魅力（以及用各种方式确保下属的忠诚），与古代中国传统权威相类似的是，最高统帅依然拥有最高级别的专断权力。

政治上“传统权威”和“卡理斯玛权威”的合法性基础都是统一，任何多元的发展都有可能会摧毁它，因而对于社会文化氛围也有进行政治教化的必要，即运用一些宣传手段使得社会意识形态也尽量一统，将主流思想之外的思想贬斥为异端，这类教化已经进行了数千年，集权及极权思想也会经由教化而传导到家庭中，而在一个家庭中“父亲”的身份就赋予了那个人最高级别的专断权力。

如果说社会上存在不平等，这里简单描述为压迫者和被压迫者的关系，假如两方都需要经历觉醒才会认可双方应平等的话，那么被压迫者的觉醒可能比压迫者的觉醒要来得更早或者更容易实现一些。想想历史上封建专制的政权，基本都是暴力推翻的，没有掌控了权力的人愿意自动让渡权力。

当然，这种政治上一统体制造就家庭中暴君思维的逻辑，也只是说得通而已。

**Reply by botsp** [reply]
应该是人伦纲常将封建的政治伦理贯穿到到家庭生活吧，反对封建家长权威也是一个挺久的革命主题了。所以感觉看到最后那段父与子的对话、就很熟悉很熟悉。

"卡理斯玛权威"，也有一个被压迫者的角度。有时候并不一定是真的存在有多大的魅力与权威，而是他身边有一大堆幕僚，认为塑造这样一个权威式领袖对于治理这样一个庞大而复杂的帝国是灵丹妙药。
将一只猪映射为一个强大而睿智的神像，也是香火鼎盛。一盘散沙的民众总需要一个伟大领袖来凝聚共识团结一致。

有时候想想，可能他们唯一的理论基础就是这个"卡理斯玛权威"吧，除了塑造神像、其它的一无所成，哈哈。  春节快乐~

**Reply by earfanfan** [reply]
其实地铁上那段父子的对话还有后续，后来那个儿子又问“那要是以后我摸别人是不是也不用经过别人同意？”，那个父亲沉默了，再后来我就下车了。

也并不是一无所成的，经济发展也还是一个奇迹，只不过现在看量的发展是以牺牲质的发展为代价得来的。祝你和你的家人们春节快乐too~

**Reply by botsp** [reply]
机智，哈哈，小孩子棒
